tests/testthat/test_wt_corr.R

```diff
@@ -1,6 +1,6 @@
 context("")
 test_that("the weighted correlation is", {
-  expect_equal(wt.corr(1:8, 2:9, rep(0, 8)), 1, tolerance=1e-4)
```

Comment:
We no longer need the `tolerance` here, but let's keep it in mind for other test cases.

Reply:
I remember learning this when I was first introduced to testthat package. There is a function expect_identical() as well. I am not sure what the tolerance level is preset.